src/video/emscripten/SDL_emscriptenframebuffer.c

```diff
@@ -57,18 +59,8 @@ int Emscripten_CreateWindowFramebuffer(_THIS, SDL_Window * window, Uint32 * form
     return 0;
 }
 
-int Emscripten_UpdateWindowFramebuffer(_THIS, SDL_Window * window, const SDL_Rect * rects, int numrects)
+static void Emscripten_UpdateWindowFramebufferWorker(SDL_Surface *surface)
```

Comment:
I'd make a similar comment about formatting here, but the entire file is already wrong.